docs/standards/nft-2.0/introduction.md

```diff
@@ -13,54 +13,27 @@ Check the [profile explorer](https://universalprofile.cloud/) to browse the depl
 
 :::
 
-:::info
+## Introduction  
 
-The [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts) repository on GitHub contains implementations of LSP7 and LSP8 that are [backward compatible with ERC20 and ERC721](https://github.com/lukso-network/lsp-universalprofile-smart-contracts/tree/main/contracts). But it is highly recommended to use the native ones.
+The current standards representing fungible and non-fungible tokens (NFTs) are limiting the NFTs and token industry as they lack standardization and many powering features.
 
-:::
-
-**Tokens & NFT 2.0** is a collective name for the new token and NFT standards [LSP7-DigitalAsset](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-7-DigitalAsset.md) and [LSP8-IdentifiableDigitalAsset](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md). These replace ERC20 and ERC721, which you would usually use on Ethereum.
-
->
-
-The interfaces used to interact with these standards are inspired by [EIP1155](https://eips.ethereum.org/EIPS/eip-1155), a Multi-Token standard for multiple token types (fungible, non-fungible, or other configurations).
-
-One of the main questions about NFT 2.0 is which characteristics make them the next generation of digital assets on the blockchain.
-
-**How different are they compared to traditional ERC20 tokens / ERC721 NFTs?**
-
-![LSP7 and LSP7 diagram](/img/standards/lsp7-lsp8-diagram.png)
-
-## How Tokens & NFT 2.0 are different?
-
-### Unlimited Metadata
-
-:::info
-
-See the [LSP4 - Digital Asset Metadata standard](./LSP4-Digital-Asset-Metadata.md) for more information.
-
-:::
-
-The existing tokens and NFTs standards do not offer a standard way to attach information to the contracts themselves. Such information (= metadata) is crucial to make each token or NFT descriptive and customized.
-
-Consider the current ERC20 and ERC721 standards as an example. These standards only define a `name`, `symbol`, and `tokenURI`. But how about if we would like to attach more specific data to the asset? (_e.g., icon, asset creator(s), token utility or motive, community behind it_).
+The lack of standardization can be seen by having each NFT or token standard implementing function names differently, which leads to have contracts supporting hundred of non-interoperable functions.
```

Comment:
Great to mention about non-interroperability. I think it's important and not mentioned enough 👍 

I am suggesting a bit of rewording + extra additions.

```suggestion
The lack of standardization can be seen through each NFT or token standard implementing its own interface with its own function names. This leads to having multiple token contracts deployed on networks, implementing different interfaces, and being non-interoperable between each other.
```